test/cpp/qps/qps_json_driver.cc

```diff
@@ -145,7 +150,8 @@ static double SearchOfferedLoad(double initial_offered_load,
                                 bool* success) {
   std::cerr << "RUNNING SCENARIO: " << scenario->name() << "\n";
   double current_offered_load = initial_offered_load;
-  double current_cpu_load = GetCpuLoad(scenario, current_offered_load, success);
+  double current_cpu_load =
```

Comment:
TODO: Change the spacing back to the way it was once I've confirmed all the tests pass